File: 351001/orsik/publisher/src/main/java/com/sergey/orsik/dto/response/AuthTokenResponseTo.java
package com.sergey.orsik.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthTokenResponseTo {
    private String access_token;
    private String type_token;
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/dto/response/AuthTokenResponseTo.kt
package com.example.distcomp.dto.response

import com.fasterxml.jackson.annotation.JsonProperty

data class AuthTokenResponseTo(
    @field:JsonProperty("access_token")
    val accessToken: String,
    @field:JsonProperty("token_type")
    val tokenType: String = "Bearer"
)
